partcad/src/partcad/runtime_python_conda.py

```
@@ -83,8 +83,11 @@ def verify_conda(self):
                     encoding="utf-8",
                 )
                 stdout, stderr = p.communicate()
-                if not stderr is None and stderr.strip() != "":
-                    pc_logging.warning("conda venv check error: %s" % stderr)
+                if p.returncode != 0:
+                    if stderr is not None and stderr.strip() != "":
+                        pc_logging.warning("conda venv check error: %s" % stderr)
+                    else:
+                        pc_logging.warning(f"conda venv check error")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)